lib/Backends/OpenCL/OpenCL.h

```diff
@@ -215,6 +215,23 @@ class OCLBackend final : public BackendUsingGlowIR {
   /// @}
 };
 
+namespace runtime {
+struct OpenCLDeviceBindings : DeviceBindings {
+  OpenCLDeviceBindings() : DeviceBindings(BackendKind::OpenCL) {}
```

Comment:
An OpenCLDeviceBindings needs to have all the fields below initialized, right?  If so make them part of the ctor.